### DISCUSSION TITLE
Can you add support to F3800 with Home Smart Panel

### DISCUSSION THREAD

**Discussion by sierralx** [thomluther/ha-anker-solix#122] [Ideas]
Has anyone been able to add the F3800 with the Home Smart Panel system to Home Assistant?

**Comment by thomluther** [comment]
I think this is mostly discussed in [this issue](https://github.com/thomluther/ha-anker-solix/issues/96) and the [api library repo issue](https://github.com/thomluther/anker-solix-api/issues/117) why it is not working
